myapp/spec/system/tasks_spec.rb

```diff
@@ -57,6 +57,13 @@
           expect(page).to have_no_content first_task.description
         end
       end
+
+      context 'アクセス時' do
+        it 'ページングのデフォルト表示の件数が正しいこと' do
+          create_list(:task, 3)
+          expect(all('tbody tr').size).to eq(5)
+        end
+      end
```

Comment:
Maybe another test for interacting with pagination?

Reply:
You're right.
I added some cases for testing paging each function.

https://github.com/Fablic/training/pull/1309/commits/d9c9cf69df23e774a7cab8840f57e23b035dcd8f